diff --git a/flatflow/ops/graph_generated.h b/flatflow/ops/graph_generated.h
index 7bfa271..aeaff0d 100644
--- a/flatflow/ops/graph_generated.h
+++ b/flatflow/ops/graph_generated.h
@@ -94,7 +94,7 @@ struct TensorMetadataBuilder {

 inline ::flatbuffers::Offset<TensorMetadata> CreateTensorMetadata(
     ::flatbuffers::FlatBufferBuilder &_fbb,
-    flatflow::ScalarType dtype = flatflow::ScalarType_FLOAT32,
+    flatflow::ScalarType dtype = flatflow::ScalarType::FLOAT32,
     ::flatbuffers::Offset<::flatbuffers::Vector<const flatflow::SymInt *>> shape = 0) {
   TensorMetadataBuilder builder_(_fbb);
   builder_.add_shape(shape);
@@ -104,7 +104,7 @@ inline ::flatbuffers::Offset<TensorMetadata> CreateTensorMetadata(

 inline ::flatbuffers::Offset<TensorMetadata> CreateTensorMetadataDirect(
     ::flatbuffers::FlatBufferBuilder &_fbb,
-    flatflow::ScalarType dtype = flatflow::ScalarType_FLOAT32,
+    flatflow::ScalarType dtype = flatflow::ScalarType::FLOAT32,
     const std::vector<flatflow::SymInt> *shape = nullptr) {
   auto shape__ = shape ? _fbb.CreateVectorOfStructs<flatflow::SymInt>(*shape) : 0;
   return flatflow::CreateTensorMetadata(
@@ -174,7 +174,7 @@ struct NodeBuilder {

 inline ::flatbuffers::Offset<Node> CreateNode(
     ::flatbuffers::FlatBufferBuilder &_fbb,
-    flatflow::Operator target = flatflow::Operator__SOFTMAX,
+    flatflow::Operator target = flatflow::Operator::_SOFTMAX,
     ::flatbuffers::Offset<::flatbuffers::Vector<::flatbuffers::Offset<flatflow::TensorMetadata>>> args = 0,
     ::flatbuffers::Offset<flatflow::TensorMetadata> meta = 0) {
   NodeBuilder builder_(_fbb);
@@ -186,7 +186,7 @@ inline ::flatbuffers::Offset<Node> CreateNode(

 inline ::flatbuffers::Offset<Node> CreateNodeDirect(
     ::flatbuffers::FlatBufferBuilder &_fbb,
-    flatflow::Operator target = flatflow::Operator__SOFTMAX,
+    flatflow::Operator target = flatflow::Operator::_SOFTMAX,
     const std::vector<::flatbuffers::Offset<flatflow::TensorMetadata>> *args = nullptr,
     ::flatbuffers::Offset<flatflow::TensorMetadata> meta = 0) {
   auto args__ = args ? _fbb.CreateVector<::flatbuffers::Offset<flatflow::TensorMetadata>>(*args) : 0;
